druid_20/Makefile

```diff
@@ -0,0 +1,12 @@
+NAME = druid
+TAG  = $(DRUID_VSN)
+IMAGE = pubnative/$(NAME):$(TAG)
```

Comment:
I've used the same repo as the druid, used by the data team. They have different versions though (v0.11 something vs. 0.20.0) and neither uses 'latest' to my knowledge. Is it okay, or it's better to use a separate docker repo?

// CC: @pubnative/data-engineering @mpolednik

Reply:
I see you are not loading the extensions used by data: mysql and parquet. Also, you have a different entry point. This would mean it won't work if we update to that image on the data side. Should we change the name to `druid_backend` and `druid_data` for our future images?

Reply:
It looks like all core extensions are already there:
```
[root@7d1ce9146b28 druid]# cd extensions/
[root@7d1ce9146b28 extensions]# ls
druid-avro-extensions             druid-lookups-cached-global
druid-azure-extensions            druid-lookups-cached-single
druid-basic-security              druid-orc-extensions
druid-bloom-filter                druid-pac4j
druid-datasketches                druid-parquet-extensions
druid-ec2-extensions              druid-protobuf-extensions
druid-google-extensions           druid-ranger-security
druid-hdfs-storage                druid-s3-extensions
druid-histogram                   druid-stats
druid-kafka-extraction-namespace  mysql-metadata-storage
druid-kafka-indexing-service      postgresql-metadata-storage
druid-kerberos                    simple-client-sslcontext
druid-kinesis-indexing-service
```

about different endpoint: I've checked [the one used in data](https://github.com/pubnative/docker-images/blob/master/druid/base/docker-entrypoint.sh) and it seems like you can easily switch to use standard `./bin/run-druid`. The only thing it doesn't do is changing a user, however I believe [this part](https://github.com/pubnative/docker-images/blob/master/druid/base/docker-entrypoint.sh#L22) doesn't work in the old image.

I'll add a user switch command, once it there I believe the functionality will be on pair with the old druid image and you technically can use this image for your deployments. This wouldn't be just a replace of image in manifests, it would require some config rework, but I guess that would be required anyway for upgrading from 0.12 to 0.20.

Reply:
Cool. Thank you. I'm not sure if you know, but after the image is created, we have the manifests on `data-tasks/eks/druid`. You can create your deployments from there once you test it locally.

Reply:
Created a user+group.

>  I'm not sure if you know, but after the image is created, we have the manifests on data-tasks/eks/druid. You can create your deployments from there once you test it locally.

Thanks, will check. Our plan was to create a single pod druid initially, just to enable experimentation.